examples/components/input/form_builder.html

```diff
@@ -15,8 +15,7 @@
     <form [ngFormModel]="projectForm">
 
       <md-input-container class="md-block">
-        <label for="description">Description</label>
-        <input md-input ngControl="description" id="description" [(value)]="model.description">
+        <label for="description">Description</label>[(value)]="model.description">
```

Comment:
Ahh yes I’ve removed the input  of the description. probably should look like.

 <label for="description">Description</label>
        <input md-input type="text" ngControl="description" id="description" />

Also, I think the MdInput would work better if the [(value)] binding was deprecated and replaced with Angular’s inbuilt [(NgModel)]. 

Binding to NgModel instead of the custom [(value)] I think would solve a few of the problems (i.e. #100) 

Id happily rewrite the entire MdInput if you thought it was the right direction.. sorry about the unnecessary PR Includes and the mistake is there something I can read r.e. contributing?

> On 7 Apr 2016, at 1:47 PM, Grégory Copin <notifications@github.com> wrote:
> 
> In examples/components/input/form_builder.html <https://github.com/justindujardin/ng2-material/pull/154#discussion_r58817474>:
> 
> > @@ -15,8 +15,7 @@
> >      <form [ngFormModel]="projectForm">
> >  
> >        <md-input-container class="md-block">
> > -        <label for="description">Description</label>
> > -        <input md-input ngControl="description" id="description" [(value)]="model.description">
> > +        <label for="description">Description</label>[(value)]="model.description">
> I think you've unfortunetaly remove something here.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/justindujardin/ng2-material/pull/154/files/77931d0e60bed698505a90a888f489dbfa3b55e3#r58817474>